prow/config/prow-config-documented.yaml

```diff
@@ -0,0 +1,575 @@
+branch-protection:
```

Comment:
Something is buggy here in genyaml, this does have comments in the source. It works when directly generating for the `BranchProtection` type. I would like to defer looking into this to a follow-up though as whatever it doesn't get introduced by this PR.